play/plugin/src/main/kotlin/com/github/triplet/gradle/play/tasks/internal/PlayApiService.kt

```diff
@@ -37,11 +41,17 @@ internal abstract class PlayApiService @Inject constructor(
     private val editIdFileAndFriends
         get() = listOf(editIdFile, editIdFile.marked("commit"), editIdFile.marked("skipped"))
 
+    var buildFailed = false
+        private set
+
+    var cleanupAtClose = true
```

Comment:
This isn't needed anymore right? Can just be replaced with `shouldCommit` down in `fun close`.

Reply:
Very true, it's gone now 💪